frontend/src/home/Home.tsx

```diff
@@ -453,6 +462,37 @@ class HomeComponent extends React.Component<Props, HomeState> {
     }
   }
 
+  renderTransfer() {
+    // If a user is currently logged in, wait until plans are fetched to render
+    if (this.props.isLoggedIn) {
```

Comment:
I think we talked about this refactor, but this case can be simplified to this.props.isLoggedIn && ! this.state.fetchedPlan 
-> SpinnerWrapper, else render transferCredits